test/pokemon-legality-checker.spec.js

```diff
@@ -24,8 +24,22 @@ describe('pokemon-legality-checker', () => {
         const parsedFile = pk6parse.parseFile(`${__dirname}/pk6/illegal/${filename}`, {parseNames: true});
         const check = legalityCheck(parsedFile);
         expect(check.isLegal).to.be.false(`Incorrectly classifies the illegal Pokémon ${filename} as legal`);
-        expect(check.reason).to.be.a('string');
+        expect(check.errors.length).to.not.be.equal(0);
+        expect(check.errors[0].field).to.be.a('string');
```

Comment:
Idea: Add another assertion:

```js
check.errors.forEach(error => {
  expect(parsedFile).to.have.ownProperty(error.field);
});
```

This will make sure that we don't end up with any typos in the field for a given rule.